Context/ChefsterDbContext.cs

```diff
@@ -11,6 +11,14 @@ public class ChefsterDbContext(DbContextOptions<ChefsterDbContext> options) : Db
     public DbSet<ConsiderationsModel> Considerations { get; set; }
     public DbSet<PreviousRecipeModel> PreviousRecipes { get; set; }
 
+    string? endpoint = Environment.GetEnvironmentVariable("MYSQL_ENDPOINT");
+    string? db = Environment.GetEnvironmentVariable("MYSQL_DB");
+    string? username = Environment.GetEnvironmentVariable("MYSQL_USERNAME");
+    string? password = Environment.GetEnvironmentVariable("MYSQL_PASSWORD");
```

Comment:
Should we pass in the configuration here via dependency injection to get the variables?